components/content/content.tsx

```diff
@@ -1,20 +1,46 @@
 import React from "react";
 import './content.scss';
 
-interface Link {
+type Link = {
```

Comment:
Er type bedre enn interface i dette tilfellet? Isåfall, hvorfor?

Reply:
det er masse meninger om det, men i kontekst av React og props så er det hipp som happ. Jeg bruker som regel `type` fordi det er kortere å skrive 🤷‍♂